postprocessing/comstockpostproc/comstock.py

```diff
@@ -2571,7 +2643,7 @@ def export_data_and_enumeration_dictionary(self):
             col_enums = []
             if col_def['data_type'] == 'string':
                 str_enums = []
-                for enum in self.data.select(col).unique().to_series().to_list():
+                for enum in self.data.columns:
```

Comment:
The original code isn't looping through column names, it's getting the unique set of values across all rows within the column.